internal/storage/storage_handle.go

```diff
@@ -150,6 +167,18 @@ func NewStorageHandle(ctx context.Context, clientConfig storageutil.StorageClien
 		return
 	}
 
+	// TODO: We will implement an additional check for the HTTP control client protocol once the Go SDK supports HTTP.
+	if clientConfig.EnableHNS {
+		clientOpts, err := createClientOptionForGRPCClient(&clientConfig)
+		if err != nil {
+			return nil, fmt.Errorf("Error in getting clientOpts for gRPC client: %w", err)
```

Comment:
nit: We should follow this (https://go.dev/wiki/CodeReviewComments#error-strings) while writing error message. Please update at all the places in this change list.